vignettes/connecting_to_sql.Rmd

```diff
@@ -24,6 +24,10 @@ Which you use will depend on how comfortable you are with SQL and R and if you a
 
 For more information on the other two methods, or on troubleshooting the connection between R and SQL in the Department for Education (DfE), please see the [Interacting with a SQL database section of our Analysts' Guide](https://dfe-analytical-services.github.io/analysts-guide/learning-development/r.html#interacting-with-a-sql-database-from-within-r-scripts).
 
+## Pre-requisites
```

Comment:
Will any of this work with databricks, & if so do we need to add guidance here the more people are onboarded? 
If none of it will work with databricks, should we note that somewhere?

Reply:
We definitely should add to this once databricks becomes more common, eventually these functions might not be necessary (in a couple of years time there might only be databricks and no separate SQL databases), raised #75 to cover it